Server/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+const Camps = require('../database-mongodb/Camp.js')
+
+app.get('/:campsiteId', (req, res) => {
+  console.log(req.params)
+    Camps.find({ newId: req.params.campsiteId}, (err, camps)=> {
+        if (err){
+          res.send(err)
+        }else {
```

Comment:
@alefranco1 @bastipacudan spacing (linting)

Reply:
@alefranco1 @bastipacudan Also missing semi-colons lines 13, 11, 8, 5. If you don't already have it setup, I suggest using the eslint extension in vs-code along with the AirBnB style guide config (AirBnB style guide is an FEC and SDC requirement)